app/controllers/macros_controller.rb

```diff
@@ -13,7 +14,8 @@ def author
 
   def show
     @user = User.find_by_login params[:author]
-    @macro = Macro.find_by_title params[:id], conditions: { user_id: @user.id }, order: 'id DESC'
+    @macro = Macro.find_by_title params[:id]
```

Comment:
Hey @Georjane just going through this code, seems like removing this condition will essentially change what we are trying to fetch since we are not checking for user id so instead of finding a particular user's first record with title as "xyz" here, we will fetch the first record with this title without any relation to the user. Can you verify this once? 

I understand that this line was causing errors while executing can we instead rewrite it as - 
`@macros = Macro.find_by title: params[:id], conditions: { user_id: @user.id }, order: 'id DESC'`